staging/src/k8s.io/apiserver/pkg/endpoints/handlers/delete.go

```diff
@@ -175,13 +175,17 @@ func DeleteResource(r rest.GracefulDeleter, allowsOptions bool, scope RequestSco
 			}
 		}
 
+		scope.Trace = trace
 		transformResponseObject(ctx, scope, req, w, status, result)
 	}
 }
 
 // DeleteCollection returns a function that will handle a collection deletion
 func DeleteCollection(r rest.CollectionDeleter, checkBody bool, scope RequestScope, admit admission.Interface) http.HandlerFunc {
 	return func(w http.ResponseWriter, req *http.Request) {
+		trace := utiltrace.New("Delete " + req.URL.Path)
+		defer trace.LogIfLong(1000 * time.Millisecond)
```

Comment:
For consistency with all other requests *create/update", maybe 500ms (we use 500ms there)?

Reply:
Quick question. On the other collection operations we do 500ms but we start them later in the operation. https://github.com/kubernetes/apiserver/blob/master/pkg/endpoints/handlers/get.go#L260 for example skips any watch latency. 
I can certainly change to 500ms but I am wondering if that may trigger to often on collection requests which are expected to take longer.